docs/apache-airflow-providers-docker/index.rst

```diff
@@ -49,7 +49,7 @@
     :maxdepth: 1
     :caption: Resources
 
-    Example DAGs <https://github.com/apache/airflow/tree/providers-docker/|version|/providers/tests/system/docker>
+    Example DAGs <https://github.com/apache/airflow/tree/19303ca655ad8f977642be49dc225433ffa3cef5/providers/tests/system/docker>
```

Comment:
This is not right. Docs are generated for every release. You can not hard code a specific tag.
Also see comment in https://github.com/apache/airflow/pull/44034#discussion_r1842645180